cra/task-discovery.yaml

```diff
@@ -28,6 +28,8 @@ spec:
     - name: continuous-delivery-context-environment
       description: Name of the configmap containing the continuous delivery pipeline context environment properties
       default: environment-properties
+    - name: exclude-scopes
+      description: Tag dependencies in scope as dev for the vulnerability scan
```

Comment:
exclude-scopes is a new property for the task
It should have a default otherwise existing pipelines (not the samples one) that are using this task will be broken